src/main/sensors/sensors.h

```diff
@@ -40,7 +40,7 @@ typedef union flightDynamicsTrims_u {
     flightDynamicsTrims_def_t values;
 } flightDynamicsTrims_t;
 
-#define CALIBRATING_GYRO_CYCLES             1000
+#define CALIBRATING_GYRO_CYCLES             1001
```

Comment:
What is this change for?

Reply:
none special
I just like primes.

Reply:
That's not a good reason for this change. At 1000, readers will assume that this was arbitrarily chosen within the range that is possible, because it's a 'round' number in the decimal system. At 1001, readers will assume that there is a reason for it not being 1000, and left wondering why.